dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/TrackerObjectsMapper.java

```diff
@@ -400,7 +400,8 @@ private TrackerObjectsMapper() {
     return dbNote;
   }
 
-  // TODO: should we remove this method when we will separate the event tables?
+  // To remove when refactoring ProgramNotificationInstanceController
+  @Deprecated
```

Comment:
set forRemoval true if this is going to be removed soon

Reply:
This refactor is not part of this epic.
I would like to remove it soon but I am not sure when we will do so.